apps/system/js/accessibility.js

```diff
@@ -169,6 +169,9 @@
                   this.toggleShade(aValue, !aValue);
                 }
                 this.screen.classList.toggle('screenreader', aValue);
+                this.announceScreenReaderState()
```

Comment:
So lets check aValue against oldValue here and only announce if they are different?